src/truncate.jl

```diff
@@ -80,6 +80,28 @@ logccdf(d::Truncated, x::Int) = x <= d.lower ? 0.0 :
                                  x >= d.upper ? -Inf :
                                  log(d.ucdf - cdf(d.untruncated, x)) - d.logtp
 
+# Callback functions for generic type
+
+pdf(d::Truncated, x::Real) = d.lower <= x <= d.upper ? pdf(d.untruncated, x) / d.tp : 0.0
```

Comment:
Instead of hard-coded `Float64` literals, the return type should be inferred from the inputs. This might yet be the case for all of the package but we are working towards it. You should be to find some example in the source where the input types are used to compute the correct return type.